src/GovUk.Education.ExploreEducationStatistics.Common/Model/Chart/ChartDataSet.cs

```diff
@@ -34,5 +34,8 @@ public class ChartDataSetConfiguration
 
         [JsonConverter(typeof(StringEnumConverter))]
         public ChartLineStyle? LineStyle;
+
+        [JsonConverter(typeof(StringEnumConverter))]
+        public ChartInlinePosition? InlinePosition;
```

Comment:
We don't need this here as `ChartDataSetConfiguration` has been deprecated for a while and isn't used anymore. We should get rid of it at some point but there's potentially a bunch of old charts from way back when that have this configuration (would need a migration to fix).

Reply:
Ok, sorted now.